src/plugin.js

```diff
@@ -173,33 +175,34 @@ const onPlayerReady = (player, options) => {
         : <div class="vjs-errors-message">${player.localize(error.message)}</div>
         </div>`;
     }
-
     display = player.errorDisplay;
```

Comment:
I think this line should change to `display = player.getChild('errorDisplay');` since we are trying to encourage using the "proper" API.